server/odc-test/src/main/java/com/oceanbase/odc/test/database/TestDBConfigurations.java

```diff
@@ -150,6 +150,11 @@ private void createTestDatabasesAndUpdateConfig(TestDBType type, TestDBConfigura
                 sql = new StringBuilder("GRANT SYSDBA, RESOURCE, CREATE SESSION TO ").append(username);
                 stmt.execute(sql.toString());
                 log.info("grant sysdba to new created user, username: {}", username);
+                // Although the above code has granted sysdba role to the new user, the connection created with the
+                // new user does not use the sysdba role, so the following grant statement is required
+                sql = new StringBuilder("GRANT ALL PRIVILEGES TO ").append(username);
```

Comment:
If grant all privileges, can we delete the above grant sql?

Reply:
In fact, sysdba is granted more privileges than all privileges and is recommended to be reserved

Reply:
removed above code